utils/fixed_decimal/src/compact.rs

```diff
@@ -92,10 +136,6 @@ fn test_compact_syntax_error() {
         pub expected_err: Option<Error>,
     }
     let cases = [
-        TestCase {
```

Comment:
nit: probably add some testcases that ensure it works now

Reply:
They are in the doc tests (and they even caught a hint bug); the `#[test]` only covers errors.

Reply:
ah neat